### PR TITLE
wazirx.handleMessage - replace.inArray with string.includes

### PR DESCRIPTION
**Pull request by samgermain**
```
% py wazirx watchTicker BTC/USDT 
Python v3.9.6
CCXT v4.2.98
wazirx.watchTicker(BTC/USDT)
{'ask': 63998.0,
 'askVolume': None,
 'average': 62392.5,
 'baseVolume': None,
 'bid': 63680.0,
 'bidVolume': None,
 'change': -825.0,
 'close': 61980.0,
 'datetime': '2024-04-17T04:09:00.000Z',
 'high': 64199.0,
 'info': {'E': 1713326940000,
          'T': 'SPOT',
          'U': 'usdt',
          'a': '63998.0',
          'b': '63680.0',
          'c': '64073.0',
          'h': '64199.0',
          'l': '61980.0',
          'o': '62805',
          'q': '0.53733',
          's': 'btcusdt',
          'u': 'btc'},
 'last': 61980.0,
 'low': 61980.0,
 'open': 62805.0,
 'percentage': -1.3135896823501314,
 'previousClose': None,
 'quoteVolume': 0.53733,
 'symbol': 'BTC/USDT',
 'timestamp': 1713326940000,
 'vwap': None}
 ```

 ```
 % py wazirx watchMyTrades          
Python v3.9.6
CCXT v4.2.98
wazirx.watchMyTrades()
[{'amount': 5.0,
  'cost': 2.5488499999999994,
  'datetime': '2024-04-17T04:14:37.000Z',
  'fee': {'cost': 0.0050977, 'currency': 'USDT', 'rate': None},
  'fees': [{'cost': 0.0050977, 'currency': 'USDT', 'rate': None}],
  'id': '416277681',
  'info': {'E': 1713327277000,
           'S': 'bid',
           'U': 'usdt',
           'a': 12454281,
           'b': 13546454,
           'c': '8da31a59-c4e4-440d-9bad-5d2dfbecc2a6',
           'f': '0.0050977',
           'ga': '0.0',
           'gc': 'usdt',
           'm': True,
           'o': 4359536415,
           'p': '0.5097699999999999',
           'q': '5.0',
           's': 'xrpusdt',
           't': 416277681,
           'w': '2.54885'},
  'order': '4359536415',
  'price': 0.50977,
  'side': 'bid',
  'symbol': 'XRP/USDT',
  'takerOrMaker': 'maker',
  'timestamp': 1713327277000,
  'type': None}]
 ```

### CODE REVIEW

File: ts/src/pro/wazirx.ts [hunk 1]
```diff
@@ -774,7 +774,7 @@ export default class wazirx extends wazirxRest {
         };
         const streams = Object.keys (streamHandlers);
         for (let i = 0; i < streams.length; i++) {
-            if (this.inArray (streams[i], stream)) {
```
[the remark this answers is not in the document]
Comment (samgermain) [reply ts/src/pro/wazirx.ts:777]: do you know what we do instead?

**Comment by fxxxfun** [comment]
i do not know yet!

**Comment by carlosmiei** [comment]
@samgermain can you check why this is not working (the unified response is not resolved)

```
 n wazirx watchTicker "BTC/INR" --verbose
Debugger attached.
2024-04-17T08:11:00.562Z
Node.js: v18.18.0
CCXT v4.2.98
wazirx.watchTicker (BTC/INR)
2024-04-17T08:11:15.541Z connecting to wss://stream.wazirx.com/stream
2024-04-17T08:11:16.538Z onUpgrade
2024-04-17T08:11:16.539Z onOpen
2024-04-17T08:11:16.564Z onMessage { data: { timeout_duration: 1800 }, event: 'connected' }
2024-04-17T08:11:16.567Z sending { event: 'subscribe', streams: [ '!ticker@arr' ] }
2024-04-17T08:11:16.805Z onMessage { data: { streams: [ '!ticker@arr' ] }, event: 'subscribed', id: 0 }
2024-04-17T08:11:20.144Z onMessage {
  data: [
    {
      E: 1713341476000,
      T: 'SPOT',
      U: 'inr',
      a: '290337.9',
      b: '287500.0',
      c: '290344.0',
      h: '292999.2',
      l: '286622.5',
      o: '292479.5',
      q: '21.4627',
      s: 'ethinr',
      u: 'eth'
    },
    {
      E: 1713341476000,
      T: 'SPOT',
      U: 'inr',
      a: '12.0585',
      b: '11.5033',
      c: '11.5345',
      h: '12.1',
      l: '11.5062',
      o: '11.6827',
      q: '7915.0',
      s: 'reqinr',
      u: 'req'
    },
    {
      E: 1713341477000,
      T: 'SPOT',
      U: 'inr',
      a: '50868.81',
```

**Comment by samgermain** [comment]
> @samgermain can you check why this is not working (the unified response is not resolved)
> 
> ```
>  n wazirx watchTicker "BTC/INR" --verbose
> Debugger attached.
> 2024-04-17T08:11:00.562Z
> Node.js: v18.18.0
> CCXT v4.2.98
> wazirx.watchTicker (BTC/INR)
> 2024-04-17T08:11:15.541Z connecting to wss://stream.wazirx.com/stream
> 2024-04-17T08:11:16.538Z onUpgrade
> 2024-04-17T08:11:16.539Z onOpen
> 2024-04-17T08:11:16.564Z onMessage { data: { timeout_duration: 1800 }, event: 'connected' }
> 2024-04-17T08:11:16.567Z sending { event: 'subscribe', streams: [ '!ticker@arr' ] }
> 2024-04-17T08:11:16.805Z onMessage { data: { streams: [ '!ticker@arr' ] }, event: 'subscribed', id: 0 }
> 2024-04-17T08:11:20.144Z onMessage {
>   data: [
>     {
>       E: 1713341476000,
>       T: 'SPOT',
>       U: 'inr',
>       a: '290337.9',
>       b: '287500.0',
>       c: '290344.0',
>       h: '292999.2',
>       l: '286622.5',
>       o: '292479.5',
>       q: '21.4627',
>       s: 'ethinr',
>       u: 'eth'
>     },
>     {
>       E: 1713341476000,
>       T: 'SPOT',
>       U: 'inr',
>       a: '12.0585',
>       b: '11.5033',
>       c: '11.5345',
>       h: '12.1',
>       l: '11.5062',
>       o: '11.6827',
>       q: '7915.0',
>       s: 'reqinr',
>       u: 'req'
>     },
>     {
>       E: 1713341477000,
>       T: 'SPOT',
>       U: 'inr',
>       a: '50868.81',
> ```

It resolves when I run it

**Comment by samgermain** [comment]
> @samgermain Can you please double-check it? I ran it again and still does not work

It looks like BTC/INR isn't being returned very often from the exchange, the response is only resolved if the that markets ticker is returned, I tried with `TRX/INR` and it was being resolved. The right symbol to choose probably depends on time, so maybe check the verbose output and choose a ticker that's commonly being returned. It's working, the exchange just isn't returning the information you're looking for

**Comment by carlosmiei** [comment]
DEMO

```
 p wazirx watchOHLCV BTC/INR           
Python v3.11.7
CCXT v4.2.98
wazirx.watchOHLCV(BTC/INR)
[[1713534420000,
  5968481.0,
  5968481.0,
  5968481.0,
  5968481.0,
  0.0010500000000000002]]
```

**Pull request merged**